src/submit.py

```diff
@@ -60,7 +61,43 @@ async def _create_submission_commit(repo_dir: pathlib.Path):
         stdout=asyncio.subprocess.PIPE,
         stderr=asyncio.subprocess.STDOUT,
     )
-    return pop_process.communicate()
+    return await pop_process.communicate()
+
+
+async def git_clone_instructions(repo_dir: pathlib.Path):
+    process = await asyncio.subprocess.create_subprocess_exec(
+        "hostname",
+        "-I",
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.STDOUT,
+    )
+    stdout, _ = await process.communicate()
+
+    click.confirm(
+        textwrap.dedent(
+            f"""
+            This task does not have internet access, so it can't automatically push to the
+            remote github repo.
+
+            Please copy the repo (which is in `/home/agent`) to your local machine and push
+            to github from there:
+
+                mkdir baseline-solution
+                git clone agent@{stdout.decode().strip()}:/home/agent.
```

Comment:
This will certainly not work. Connections to task environments go through a jumphost

Reply:
I opened https://github.com/evals-sandbox/baseline-ops/pull/151 with the starting point for what we'd need to be able to provide baseliners with exact, explicit instructions.

Reply:
this was from the [official instructions](https://docs.google.com/document/d/1D2JZpCqVFaj0yLYuBO7uxdEvKmshfHtDlCt1rpROvgA/edit?tab=t.bgyd29n2ksmu), which means that's also wrong (it happens...). It should now give copy+pasteable instructions (other than where the key lives, of course). I considered also having the key be part of it, but that seems excessive?